docs/validate-github-event.yaml

```diff
@@ -24,7 +24,7 @@ spec:
         cat <<EOF | python
         import hashlib, os, hmac, json
         secret = bytes(os.environ.get('GithubSecret'), 'utf-8')
-        payload = bytes("$(inputs.params.EventBody)",'utf-8')
+        payload = bytes('$(inputs.params.EventBody)','utf-8')
```

Comment:
Strange. I don't get this error. Should we use raw string instead? r'? Are you using enterprise github which sends a bit different payload?

Reply:
I'm using the github enterprise but I'm not sure if it makes differences.  The payload is a json string for both githubs.

Reply:
Does this change cause issues with the public github?